k6ext/k6test/vu.go

```diff
@@ -139,7 +139,10 @@ func (v *VU) RunOnEventLoop(tb testing.TB, js string, args ...any) (sobek.Value,
 func (v *VU) RunAsync(tb testing.TB, js string, args ...any) (sobek.Value, error) {
 	tb.Helper()
 
-	return v.RunOnEventLoop(tb, fmt.Sprintf("(async function() { "+js+" })();", args...))
```

Comment:
Hoping to fix https://github.com/grafana/xk6-browser/actions/runs/10773846755/job/29877212064 with this change 🤞

Reply:
The number of args don't match to the number of values, I guess.

Reply:
Never seen this issue before 🤷 Seems to only occur with the tip.

Reply:
Related (IDK :))?

* https://github.com/argoproj/argo-cd/issues/19772
* https://cs.opensource.google/go/x/tools/+/refs/tags/v0.24.0:go/analysis/passes/printf/printf.go;l=502-534
* https://code.waarp.fr/apps/gateway/gateway/-/blob/master/.golangci.yml#L841-844

Reply:
Right, i get it, it's a new change in tip, that's why it's failing on the build step 👍

Reply:
I still don't understand why the original change wasn't building, but the new change seems to have solved it.